### PR TITLE
Release 1.7.0

### PR DESCRIPTION
**Pull request by obierlaire**
Related to https://github.com/elastic/ecctl/issues/531
Relase 1.7.0

### CODE REVIEW

File: docs/release_notes/v1.7.0.adoc [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+[id="{p}-release-notes-v1.7.0-bug-fixes"]
+==== Bug fixes
+
+// TODO: FILL OR REMOVE
+
+[float]
+[id="{p}-release-notes-v1.7.0-breaking-changes"]
+==== Breaking changes
+
+// TODO: FILL OR REMOVE
```
Comment (alaudazzi) [inline docs/release_notes/v1.7.0.adoc:40]: Shall we keep these sections even if they are empty?

**Review by alaudazzi — APPROVED** [review]
LGTM, except for one comment.

File: docs/release_notes/v1.7.0.adoc (hunk 1 above)
Comment (obierlaire) [reply docs/release_notes/v1.7.0.adoc:40]: @marclop @elastic/cloud-delivery what should we do?

Comment (marclop) [reply docs/release_notes/v1.7.0.adoc:40]: should remove anything that's empty.

File: docs/ecctl-release-notes.asciidoc [hunk 2]
```diff
@@ -26,3 +27,4 @@ include::release_notes/v1.0.0.adoc[]
 include::release_notes/v1.0.0-beta3.adoc[]
 include::release_notes/v1.0.0-beta2.adoc[]
 include::release_notes/v1.0.0-beta1.adoc[]
+include::release_notes/v1.7.0.adoc[]
```
Comment (marclop) [inline docs/ecctl-release-notes.asciidoc:30]: This guy should be reordered at the top of the `include::` list, since otherwise it'll show up at the bottom and it's not what we want.

File: docs/release_notes/v1.7.0.adoc [hunk 3]
```diff
@@ -0,0 +1,48 @@
+[id="{p}-release-notes-v1.7.0"]
+=== Elastic Cloud Control (ecctl) v1.7.0
+++++
+<titleabbrev>v1.7.0</titleabbrev>
+++++
+
+<<{p}-release-notes-v1.7.0-whats-new,What's new>> | <<{p}-release-notes-v1.7.0-bug-fixes,Bug fixes>> | <<{p}-release-notes-v1.7.0-breaking-changes,Breaking changes>> | <<{p}-release-notes-v1.7.0-changelog,Changelog>>
+
+Welcome to the v1.7.0 release of {n}. This version brings new features, some breaking changes, and bug fixes.
```
Comment (marclop) [inline docs/release_notes/v1.7.0.adoc:9]: We should probably reword this to say it only includes an SDK version bump

File: docs/release_notes/v1.7.0.adoc [hunk 4]
```diff
@@ -0,0 +1,48 @@
+[id="{p}-release-notes-v1.7.0"]
+=== Elastic Cloud Control (ecctl) v1.7.0
+++++
+<titleabbrev>v1.7.0</titleabbrev>
+++++
+
+<<{p}-release-notes-v1.7.0-whats-new,What's new>> | <<{p}-release-notes-v1.7.0-bug-fixes,Bug fixes>> | <<{p}-release-notes-v1.7.0-breaking-changes,Breaking changes>> | <<{p}-release-notes-v1.7.0-changelog,Changelog>>
```
Comment (marclop) [inline docs/release_notes/v1.7.0.adoc:7]: These references aren't pointing anywhere now now that we've removed the headings, we should ammend the links otherwise the docs build might break

**Review by marclop — APPROVED** [review]

**Comment by obierlaire** [comment]
Thanks @marclop I reworded the release notes in order to be consistent with previous SDK version bumps release notes.
